template/{{.input_root_dir}}/cicd/databricks_template_schema.json

```diff
@@ -45,6 +45,24 @@
       "order": 9,
       "description": "Use Feature Store (yes) or not (no)",
       "type": "string"
+    },
+    "input_test_catalog_name": {
+      "order": 10,
+      "type": "string",
+      "description": "\nName of the Test Unity Catalog",
+      "default": "test"
+    },
+    "input_staging_catalog_name": {
+      "order": 13,
```

Comment:
Why does the order jump from 10 to 13? Also let's try to keep it in the same order as the one above, as well as push these above `cloud_specific_node_type_id`. We want to keep project name and FS last since we'll be piping in those variables from the JSON file